examples/experimental/i3s-picking/mesh-layer/mesh-layer.js

```diff
@@ -62,12 +63,36 @@ export default class MeshLayer extends SimpleMeshLayer {
     });
   }
 
+  draw({uniforms}) {
+    if (!this.state.model) {
+      return;
+    }
+
+    const {viewport} = this.context;
+    const {sizeScale, coordinateSystem, _instanced} = this.props;
+
+    this.state.model.draw({
+      uniforms: Object.assign({}, uniforms, {
+        sizeScale,
+        composeModelMatrix: !_instanced || shouldComposeModelMatrix(viewport, coordinateSystem),
```

Comment:
I am confused. We check if we should compose but we don't actually use the `calculateTransformMatrix` and the `MATRIX_ATTRIBUTES`?

Reply:
It is from base class `SimpleMeshLayer`

Reply:
I added 
```
        // Needed for PBR (TODO: find better way to get it)
        // eslint-disable-next-line camelcase
        u_Camera: this.state.model.getUniforms().project_uCameraPosition
```
to existing code